packages/phrases/src/locales/en.ts

```diff
@@ -1,31 +1,35 @@
 /* eslint-disable max-lines */
 const translation = {
-  general: {
-    placeholder: 'Placeholder',
-    skip: 'Skip',
-    next: 'Next',
-    retry: 'Try again',
-    done: 'Done',
-    search: 'Search',
-    clear_result: 'Clear Results',
-    save: 'Save',
-    save_changes: 'Save Changes',
-    loading: 'Loading...',
-    redirecting: 'Redirecting...',
-    added: 'Added',
-    cancel: 'Cancel',
-    confirm: 'Confirm',
-    check_out: 'Check out',
-    create: 'Create',
-    set_up: 'Set up',
-    customize: 'Customize',
-  },
   admin_console: {
     title: 'Admin Console',
     sign_out: 'Sign Out',
     profile: 'Profile',
     admin_user: 'Admin',
     system_app: 'System',
+    general: {
```

Comment:
I think some of the phrases above can also be put into `general`, at least `sign_out`, `profile`, `admin`

Reply:
They are not shared between pages. I prefer to keep them out of `general`.